From 16e623c70064c226c17dc91964750a1d1bcd82b1 Mon Sep 17 00:00:00 2001
From: Talley Lambert <talley.lambert@gmail.com>
Date: Tue, 11 Jul 2023 22:22:31 -0400
Subject: [PATCH] make Talley emeritus SC (#204)

As discussed elsewhere, it's time for me to step down from this role.

Thanks @napari/core-devs
---
 docs/community/team.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/community/team.md b/docs/community/team.md
index 86840e49..49c88d37 100644
--- a/docs/community/team.md
+++ b/docs/community/team.md
@@ -18,7 +18,7 @@ napari is a consensus-based community project. Anyone with an interest in the pr
 - [Wouter-Michiel Vierdag](https://github.com/napari/napari/commits?author=melonora) - [@melonora](https://github.com/melonora)
 - [Nicholas Sofroniew](https://github.com/napari/napari/commits?author=sofroniewn) - [@sofroniewn](https://github.com/sofroniewn)^
 - [Peter Sobolewski](https://github.com/napari/napari/commits?author=psobolewskiPhD) - [@psobolewskiPhD](https://github.com/psobolewskiPhD)
-- [Talley Lambert](https://github.com/napari/napari/commits?author=tlambert03) - [@tlambert03](https://github.com/tlambert03)*
+- [Talley Lambert](https://github.com/napari/napari/commits?author=tlambert03) - [@tlambert03](https://github.com/tlambert03)^

 <sub>* on the napari [steering council](https://napari.org/community/governance.html#steering-council).</sub>
